api.md

```diff
@@ -171,7 +172,115 @@ SELECT add_dimension('conditions', 'device_id', number_partitions => 2, if_not_e
+| `remote_user`        | User name on the remote data node.                    |
+| `password`           | Password on the remote data node.                     |
+| `if_not_exists`      | Do not fail if the hyper table already exists         |
+| `bootstrap_database` | Database to bootstrap.                                |
+| `bootstrap_user`     | User to used for bootstrapping.                       |
+| `bootstrap_password` | Password for user use when bootstrapping.             |
+
+##### Parameter Description
+
+* `host` - the name of the host to use for the remote data node. This
+  defaults to `'localhost'`.
+
+* `post` - Port to use for the remote data node. This defaults to the
+  default PostgreSQL port.
+
+* `database` - Name of the database to use on the remote data
+  node. Default is to use the current database in use when calling the
+  function.
+
+* `local_user` - Name of the local user. Default is to use the current
+  user.
+
+* `remote_user` - Name of the user on the remote data node. Default is
+  to use the same user as `local_user`.
+
+* `password` - Password to use on the remote data node. Defaults is to
+  not use a password.
+
+* `bootstrap_database` - The database to use when bootstrapping the
+  server as described above. The default is to use `'postgres'`.
+
+* `bootstrap_user` - User to use when bootstrapping the server as
+  described above. The default is to use the saver provided as
+  `remote_user` (or the default).
+
+* `bootstrap_password` - Password to use when bootstrapping the server
+  as described above. The default is to use the password as provided
+  as `password`.
+
+#### Returns [](add_data_node-returns)
+
+| Column          | Description                                   |
+|-----------------|-----------------------------------------------|
+| `node_name`     | Name of the data node                         |
+| `host`          | Host name for the remote data node            |
+| `port`          | Port for the remote data node                 |
+| `database`      | Name of database used on the remote data node |
+| `username`      | Local username                                |
+| `node_username` | Remote username                               |
+| `created`       |                                               |
+
+#### Sample usage [](add_data_node-examples)
+
+Let's assume that you have an existing table `conditions` and want to
+use the `time` column as the time partitioning column and the
+`location` column as the space partitioning column. You also want want
+to distribute the chunks of the table on two data nodes
+`dn1.example.com` and `dn2.example.com`.
+
+To do this, you can just type:
+
+```sql
+SELECT add_data_node('dn1', host => 'dn1.example.com');
+SELECT add_data_node('dn2', host => 'dn2.example.com');
+SELECT create_distributed_hypertable(
+    'conditions', 'time', 'location',
+	servers => '{"dn1","dn2"}'
```

Comment:
You may elide the server list here and it will make use of all servers in the database. I think this should probably be the default documented behavior. We can go into more details about servers  (should probably be `data_nodes`) in the description of `create_distributed_hypertable()`.

Reply:
Done.